modules/meeting/spec/features/meetings_new_spec.rb

```diff
@@ -50,72 +50,150 @@
+          # Setting the project reloads the page
+          # causing a StaleElementReferenceError
+          # if the execution is too quick.
+          SeleniumHubWaiter.wait
+
+          new_page.set_start_date '2013-03-28'
+          new_page.set_start_time '13:30'
+          new_page.set_duration '1.5'
+          new_page.invite(other_user)
+
+          show_page = new_page.click_create
+
+          show_page.expect_toast(message: 'Successful creation')
+
+          show_page.expect_invited(user, other_user)
 
-      it "allows creating a project and handles errors in time zone #{zone}" do
+          show_page.expect_date_time "03/28/2013 01:30 PM - 03:00 PM"
+        end
+      end
+    end
+
+    context 'without permission to create meetings' do
+      let(:permissions) { %i[view_meetings] }
+
+      it 'shows no edit link' do
+        index_page.visit!
+
+        index_page.expect_no_create_new_button
+      end
+    end
+
+    context 'as an admin' do
+      let(:current_user) { admin }
+
+      it 'allows creating meeting in a project without members' do
         index_page.visit!
 
         new_page = index_page.click_create_new
 
         new_page.set_title 'Some title'
-        new_page.set_start_date '2013-03-28'
-        new_page.set_start_time '13:30'
-        new_page.set_duration '1.5'
-        new_page.invite(other_user)
+
+        new_page.set_project project
 
         show_page = new_page.click_create
 
         show_page.expect_toast(message: 'Successful creation')
 
-        show_page.expect_invited(user, other_user)
-
-        show_page.expect_date_time "03/28/2013 01:30 PM - 03:00 PM"
+        # Not sure if that is then intended behaviour but that is what is currently programmed
+        show_page.expect_invited(admin)
       end
     end
   end
 
-  context 'without permission to create meetings' do
-    let(:permissions) { %i[view_meetings] }
+  context 'when creating a meeting from the project-specific page' do
+    let(:index_page) { Pages::Meetings::Index.new(project:) }
+
+    context 'with permission to create meetings' do
+      before do
+        other_user
+      end
+
+      ['CET', 'UTC', '', 'Pacific Time (US & Canada)'].each do |zone|
+        let(:time_zone) { zone }
+
+        it "allows creating a project and handles errors in time zone #{zone}" do
+          index_page.visit!
+
+          new_page = index_page.click_create_new
 
-    it 'shows no edit link' do
-      index_page.visit!
+          new_page.set_title 'Some title'
+          new_page.set_start_date '2013-03-28'
+          new_page.set_start_time '13:30'
+          new_page.set_duration '1.5'
+          new_page.invite(other_user)
 
-      index_page.expect_no_create_new_button
+          show_page = new_page.click_create
+
+          show_page.expect_toast(message: 'Successful creation')
+
+          show_page.expect_invited(user, other_user)
+
+          show_page.expect_date_time "03/28/2013 01:30 PM - 03:00 PM"
+        end
+      end
```

Comment:
I don't understand why those passed on the CI. They fail for me locally and I think they do so rightfully as the `participants_section` does not show up on the project scoped new form.

Reply:
Now we know :)